src/prototype.rs

```diff
@@ -953,18 +1016,6 @@ impl<'a> Tree<Item<'a>> {
     }
 }
 
-#[allow(dead_code)]
```

Comment:
I'm hesitant to delete it, as I've found it useful, but having unused code around for a long period of time is not great.